rockcraft.yaml

```diff
@@ -6,10 +6,15 @@ summary: Canonical Identity platform login UI
 description: |
   This is the Canonical Identity platform login UI used for connecting
   Ory Kratos with Ory Hydra.
-license: Apache-2.0
+license: GPL-3.0
 platforms:
   amd64:
-entrypoint: ["identity_platform_login_ui"]
+
+services:
+  login-ui:
+    override: replace
+    command: identity_platform_login_ui
+    startup: enabled
```

Comment:
Why we need to replace entrypoint with services? Will it not interfere with `services` in [pebble layer](https://github.com/canonical/identity-platform-login-ui-operator/blob/main/src/charm.py#L129)?

Reply:
@natalian98 that seems correct as per the layer specification https://github.com/canonical/pebble#layer-specification